src/patterns/utilities/position.hbs

```diff
@@ -0,0 +1,49 @@
+---
+name: Position
+notes: |
+  Includes all standard SUIT CSS position utilities:
+
+  + `u-posAbsolute` absolutely positions an element.
+  + `u-posAbsoluteCenter` absolutely positions and centre an element.
+  + `u-posFit` fits an element to the dimensions of its parent
+  + `u-posFullScreen` ixes an element over the viewport
+  + `u-posFixed` fixed positions an element.
+  + `u-posFixedCenter` fixes an element in the centre of the viewport
```

Comment:
Nit: missing a couple periods in this section. Only notice because punctuation has been so consistent elsewhere.